db-testing/src/test/java/com/splicemachine/dbTesting/functionTests/tests/lang/GeneratedColumnsHelper.java

```diff
@@ -151,6 +151,21 @@ protected void    goodStatement( Connection conn, String ddl ) throws SQLExcepti
         ps.execute();
         ps.close();
     }
+
+    /**
+     * Run a good update statement with an expected row count.
+     * @throws SQLException
+     */
+    protected void    goodUpdate( Connection conn, String update, int expectedRowCount ) throws SQLException
+    {
+        PreparedStatement    ps = chattyPrepare( conn, update );
+
+        int actualRowCount = ps.executeUpdate();
+        ps.close();
```

Comment:
Use try-with-resources to always close prepared statement

Reply:
done